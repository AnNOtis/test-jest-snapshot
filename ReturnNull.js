import React from 'react'

const ReturnNull = () => null

export default ReturnNull
