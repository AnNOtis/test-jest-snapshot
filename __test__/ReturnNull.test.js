import React from 'react';
import { shallow } from 'enzyme';

import ReturnNull from '../ReturnNull';

describe('<ReturnNull />', () => {

  it('should render', () => {
    const component = shallow(<ReturnNull />);

    expect(component).toMatchSnapshot();
  });
});
