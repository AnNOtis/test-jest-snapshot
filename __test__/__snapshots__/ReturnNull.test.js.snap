// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReturnNull /> should render 1`] = `null`;
